pandas/core/arrays/masked.py

```diff
@@ -77,6 +84,23 @@ def __getitem__(self, item):
 
         return type(self)(self._data[item], self._mask[item])
 
+    def _coerce_to_array(self, values) -> Tuple[np.ndarray, np.ndarray]:
+        raise AbstractMethodError(self)
```

Comment:
orthogranal to this PR but wondering if we even need this any more or could just use abc.abstractmethod decorator

Reply:
We generally use this instead of the decorator in the EA base class, because the decorator requires it to actually be an abstract base class. 

See 

https://github.com/pandas-dev/pandas/blob/45fee3278680e8da2724236cc93227bcb28585b1/pandas/core/arrays/base.py#L128-L131